<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GBSRemakeSlices</class>
 <widget class="QDialog" name="GBSRemakeSlices">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>815</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>815</width>
     <height>54</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>14</y>
      <width>95</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size:16px;</string>
    </property>
    <property name="text">
     <string>设置素材片段</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClose">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>2</y>
      <width>48</width>
      <height>48</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../forms/images/gbs/biz/gbs-close-window.png</normaloff>../../forms/images/gbs/biz/gbs-close-window.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_2" native="true">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>50</y>
     <width>831</width>
     <height>411</height>
    </rect>
   </property>
   <widget class="QLineEdit" name="leActivationCode">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>110</y>
      <width>451</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;
border: 1px solid  #7E7E7E;
border-radius: 8px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>  请填写你的素材片段名字(必填)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOK">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>320</y>
      <width>96</width>
      <height>59</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#00C566;
border: 1px solid white;
border-radius:10px;
font-size:20px;</string>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCancel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>320</y>
      <width>96</width>
      <height>59</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#00C566;
border: 1px solid white;
border-radius:10px;
font-size:20px;</string>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>113</x>
      <y>120</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #78828A;
font-size: 14px;</string>
    </property>
    <property name="text">
     <string>素材片段名字</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../Resources.qrc"/>
 </resources>
 <connections/>
</ui>
